packages/lit-html/src/lit-html.ts

```diff
@@ -318,20 +318,20 @@ export abstract class Directive {
 }
 
 class Template {
-  private __strings: TemplateStringsArray;
-  __element: HTMLTemplateElement;
-  __parts: Array<TemplatePart> = [];
+  private _strings: TemplateStringsArray;
+  _element: HTMLTemplateElement;
```

Comment:
`private` ?

Reply:
Since `Template` is not an exported class, and `_element` is accessed within this module but outside this class, this is really "package private" and it seems simpler to just keep it public, otherwise we'll need to cast it.

I guess if a `Template` instance was accessible by some other public typed route, even though the class itself is not exported, then it would make sense for it to be `private` and cast, but it doesn't look like that's the case.

I did find a few other misc properties that could safely become `private`.